examples/pytorch/gat/README.md

```diff
@@ -1,49 +1,49 @@
-# Introduction
```

Comment:
Looks like there are lots of non-relevant changes. Could you pick only ones related to neighbor count feature?